packages/api/src/utils/serdes.ts

```diff
@@ -66,3 +66,25 @@ export function fromU64StrOpt(u64Str: U64Str | undefined): U64 | undefined {
 export function toU64StrOpt(u64: U64 | undefined): U64Str | undefined {
   return u64 !== undefined ? toU64Str(u64) : undefined;
 }
+
+const GRAFFITI_HEX_LENGTH = 66;
```

Comment:
@wemeetagain I don't really like that the constant is defined here, do you have a better idea where we could place it?

Reply:
imo it seems fine here, next to these graffiti hex utility functions. I don't think there's anything really reusable or more broad about the graffiti hex length right now.